batch/src/main/java/com/hartwig/batch/operations/SamtoolsCramToBam.java

```diff
@@ -1,53 +1,41 @@
 package com.hartwig.batch.operations;
 
-import java.io.File;
-
 import com.hartwig.batch.BatchOperation;
 import com.hartwig.batch.input.InputBundle;
 import com.hartwig.batch.input.InputFileDescriptor;
 import com.hartwig.pipeline.ResultsDirectory;
-import com.hartwig.pipeline.calling.command.VersionedToolCommand;
-import com.hartwig.pipeline.execution.vm.Bash;
+import com.hartwig.pipeline.cram.CramAndValidateCommands;
 import com.hartwig.pipeline.execution.vm.BashStartupScript;
 import com.hartwig.pipeline.execution.vm.OutputUpload;
 import com.hartwig.pipeline.execution.vm.RuntimeFiles;
 import com.hartwig.pipeline.execution.vm.VirtualMachineJobDefinition;
 import com.hartwig.pipeline.execution.vm.VirtualMachinePerformanceProfile;
 import com.hartwig.pipeline.execution.vm.VmDirectories;
-import com.hartwig.pipeline.resource.Resource;
+import com.hartwig.pipeline.execution.vm.unix.MvCommand;
 import com.hartwig.pipeline.storage.GoogleStorageLocation;
 import com.hartwig.pipeline.storage.RuntimeBucket;
-import com.hartwig.pipeline.tools.Versions;
+
+import java.io.File;
+
+import static java.lang.String.format;
 
 public class SamtoolsCramToBam implements BatchOperation {
```

Comment:
Rename to SamtoolsBamToCram?

Reply:
Good call. Not sure why this was done this way.